### DISCUSSION TITLE
Analyzing diaPASEF data

### DISCUSSION THREAD

**Discussion by cacagong** [vdemichev/DiaNN#244] [Q&A]
Hi, 
I am trying to analyze diaPASEF data from timTOF Pro. A spectral library was generated from EasyPQP after MSFrageer search in Fragpipe. 
I am not clear on one thing. **In which step DIANN use ion mobility information for analyzing diaPASEF data?**
It seem like that there is no column or information of ion mobility  in the output spectral library from EasyPQP.

Thanks,
Dong-Gi

**Comment by vdemichev** [comment]
Hi Dong-Gi,

Please ask the FragPipe team (they usually respond very quickly) why the column with ion mobility values is absent.

Best,
Vadim

**Reply by cacagong** [reply]
Okay Vadim, Thanks!!
